pyarlo/base_station.py

```diff
@@ -270,33 +270,29 @@ def get_camera_properties(self):
     def get_camera_battery_level(self):
         """Return a list of battery levels of all cameras."""
         battery_levels = {}
-        resource = "cameras"
-        resource_event = self.publish_and_get_event(resource)
-        if resource_event:
-            cameras = resource_event.get('properties')
-            for camera in cameras:
-                serialnum = camera.get('serialNumber')
-                cam_battery = camera.get('batteryLevel')
-                battery_levels[serialnum] = cam_battery
-            return battery_levels
```

Comment:
a lot of this functionality already existed in `self.get_camera_properties`, i just reused it here.